lib/migration_generator/migration_generator.ex

```diff
@@ -491,6 +509,55 @@ defmodule AshPostgres.MigrationGenerator do
     end)
   end
 
+  defp get_dev_migrations(opts, tenant?, repo) do
+    opts
+    |> migration_path(repo, tenant?)
+    |> File.ls()
+    |> case do
+      {:error, _error} -> []
+      {:ok, migrations} -> Enum.filter(migrations, &String.contains?(&1, "_dev.exs"))
+    end
+  end
+
+  defp remove_dev_migrations(dev_migrations, tenant?, repo, opts) do
+    version = dev_migrations |> Enum.min() |> String.split("_") |> hd()
+    test_env = [env: [{"MIX_ENV", "test"}]]
+    System.cmd("mix", ["ash_postgres.rollback", "--to", version])
```

Comment:
This should use:

```elixir
Mix.Task.reenable("ash_postgres.rollback")
Mix.Task.run("ash_postgres.rollback", ["--repo", inspect(repo)])
```

(using your new `--repo` option). This also avoids needing to do anything special with `MIX_ENV` etc.

Reply:
After doing this, if I leave
```
    if tenant? do
      Enum.each(repo.all_tenants(), fn tenant ->
        opts = ["--repo", inspect(repo), "--to", version, "--prefix", tenant]
        Mix.Task.run("ash_postgres.rollback", opts)
      end)
    end
```
then it fails. I get an error
```
     ** (Ash.Error.Unknown) 
     Bread Crumbs:
       > Error returned from: AshPostgres.MultitenancyTest.Org.read
```
which comes from `repo.all_tenants()`
and there is also a log
```
     The following output was logged:
     11:30:35.118 [info] Migrations already down
     11:30:35.123 [info] Migrations already down
```

So I assume that also being done.? 🤔 I removed it then.